portal-web/test/functional/com/liferay/portalweb/macros/AnalyticsCloud.macro

```diff
@@ -43,7 +43,7 @@ definition {
 
 		Click(locator1 = "ACDXPSettings#CONNECT_DISCONNECT_BUTTON");
 
-		Alert.viewErrorMessage(errorMessage = "Your request failed to complete.");
+		Alert.viewErrorMessage(errorMessage = "An unexpected system error occurred.");
```

Comment:
Do you know if they plan on changing this error message?

Reply:
Hey Josh, I have the same question. I talked with Adriano and he said that he had put this generic message, but I will confirm the message with Déborah on Monday when she returns from vacation